nilearn/plotting/surf_plotting.py

```diff
@@ -266,9 +305,9 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
 def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
                        hemi='left', view='lateral', threshold=None,
                        alpha='auto', vmax=None, cmap='cold_hot',
-                       symmetric_cbar="auto", bg_on_data=False, darkness=1,
-                       title=None, output_file=None, axes=None, figure=None,
-                       **kwargs):
+                       colorbar=False, symmetric_cbar="auto", bg_on_data=False,
```

Comment:
maybe colorbar=True by default? it is the case for plot_stat_map

Reply:
+1